### DISCUSSION TITLE
Have option to show full module path for a type in hover

### DISCUSSION THREAD

**Discussion by starlingnic** [microsoft/pylance-release#4700] [Enhancement]
At the moment, the ability to customize the output of hints in Pylance in Visual Studio Code, including adding the full path to the object as Jedi does, unfortunately, is not provided.

The Pylance language server used by the Python extension for VS code sets its own format and output content, and does not provide a mechanism for its modification at the user's level.

Jedi:

`"python.languageServer": "Jedi"`

_Cursor at "iden" termin over:_

Hint:

```
iden = self.sockets.get(socket_id, None)

Full name: app.services.ocpp.MyChargePoint.on_meter_values.iden
```
"app.services.ocpp" - module with **app module path in Python style**
"MyChargePoint" - class
"on_meter_values" - class method
"iden" - var name

You see:

...
Full name: app.services.ocpp.MyChargePoint.on_meter_values.iden

It's ideal to have this part in tooltips for Pylance, which is based on Pyright.

**Comment by rchiodo** [comment]
Thanks for the suggestion.

**Comment by thomasrea0113** [comment]
Please add this feature! Seems simple enough from a design perspective but would greatly improve my workflow. Quite often I need to include an existing import in another file (either because I'm copying some code or extending a library class) and need to include a bunch of imported types. It's incredibly tedious to have to go each definition and/or scroll to wherever the import is just to know where that object actually comes from.
